rstudio/rhel9-python-3.9/Dockerfile

```diff
@@ -64,6 +64,11 @@ RUN chmod 1777 /var/run/rstudio-server && \
     mkdir -p /usr/share/doc/R
 COPY rstudio/rhel9-python-3.9/rsession.conf /etc/rstudio/rsession.conf
 
+# package installation 
+RUN dnf install -y libsodium-devel.x86_64 libgit2-devel.x86_64 libcurl-devel harfbuzz-devel.x86_64 fribidi-devel.x86_64 cmake "flexiblas-*"
+RUN R -e "install.packages('Rcpp')"
+RUN R -e 'install.packages(c("tidyverse", "tidymodels", "plumber", "vetiver", "devtools"))'
```

Comment:
So, we'll preinstall these after all, for everybody? Ok with me!

Reply:
Good point, is it possible to pin particular version with this command?

Reply:
And have automation to keep the versions up to date on our end? If R packages get CVS's too?

Reply:
Also - if there won't be a fixed list of package versions for the RStudio image in a given product version, does testing need to be approached differently? (i.e. if we test with the versions that are latest at the moment and then any change between releases, can we be confident in our testing?)

Reply:
We are gonna remove the R package install from this sprint 2.7. 
If you need to work more on these bits, please refer to these JIRAs for further work:
https://issues.redhat.com/browse/RHOAIENG-2647
https://issues.redhat.com/browse/RHOAIENG-2799